api/src/reconciliation/reconciliation.service.ts

```diff
@@ -199,6 +199,22 @@ export class ReconciliationService {
     });
   }
 
+  async getLatestCompReconByComponentIds(compIds: number[]) {
```

Comment:
Add `organization` here so that we can be sure we only operate on the current org.